File: src/main/java/org/intellij/tool/branch/merge/local/CommonMergeAction.java
package org.intellij.tool.branch.merge.local;

import com.intellij.openapi.actionSystem.AnActionEvent;
import com.intellij.openapi.project.DumbAwareAction;
import com.intellij.openapi.project.Project;
import org.jetbrains.annotations.NotNull;

public class CommonMergeAction extends DumbAwareAction {
    @Override
    public void actionPerformed(@NotNull AnActionEvent e) {
        Project project = e.getProject();
        if (project == null) {
            return;
        }

        CommonMergeDialog dialog = new CommonMergeDialog(project, e);
        dialog.showAndGet();
    }

}

File: src/main/java/org/intellij/tool/branch/merge/local/CommonMergeAction.kt
package org.intellij.tool.branch.merge.local

import com.intellij.openapi.actionSystem.AnActionEvent
import com.intellij.openapi.project.DumbAwareAction

class CommonMergeAction : DumbAwareAction() {
    override fun actionPerformed(e: AnActionEvent) {
        val project = e.project ?: return
        CommonMergeDialog(project, e).showAndGet()
    }
}
